programs/server/users.d/access_management.xml

```diff
@@ -0,0 +1,7 @@
+<yandex>
```

Comment:
This file is used for development - when I run `./clickhouse-server` by pressing Enter.